src/index.html

```diff
@@ -5,6 +5,7 @@
   <meta http-equiv="x-ua-compatible" content="ie=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet">
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
I thought ng2-bootstrap contained styles too, but as it turns out it is just the bootstrap features as components, sans CSS. The homepage says that for CSS, and they cited the link http://valor-software.com/ng2-bootstrap/index-bs4.html#/

I think we will want bootstrap generally, but certainly lmk what direction we want to go in!